jenkins/scripts/dynamic_worker_workflow/dev_env_integration_test.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -eu
+
+# Description:
+#   Runs the feature tests in dynamic jenkins worker
+# Usage:
+#  ./dev_env_integration_tests.sh
+
+CI_DIR="$(dirname "$(readlink -f "${0}")")"
+
+export IMAGE_OS="${IMAGE_OS:-ubuntu}"
+export REPO_ORG="${REPO_ORG:-metal3-io}"
+export REPO_NAME="${REPO_NAME:-metal3-dev-env}"
+export METAL3REPO="${METAL3REPO:-https://github.com/metal3-io/metal3-dev-env.git}"
+export METAL3BRANCH="${METAL3BRANCH:-main}"
+export CAPM3RELEASEBRANCH="${CAPM3RELEASEBRANCH:-main}"
+export BMORELEASEBRANCH="${BMORELEASEBRANCH:-main}"
+export NUM_NODES="${NUM_NODES:-1}"
+export TARGET_NODE_MEMORY="${TARGET_NODE_MEMORY:-4096}"
+export IRONIC_INSTALL_TYPE="${IRONIC_INSTALL_TYPE:-rpm}"
+export IRONIC_FROM_SOURCE="${IRONIC_FROM_SOURCE:-false}"
+export BUILD_IRONIC_IMAGE_LOCALLY=""
+
+if [[ "${IRONIC_INSTALL_TYPE}" == "source" ]]; then
+  IRONIC_FROM_SOURCE="true"
+  if [[ "${REPO_NAME}" == "ironic-image" ]]; then
+    export IRONIC_LOCAL_IMAGE="/home/${USER}/tested_repo"
+  else
+    BUILD_IRONIC_IMAGE_LOCALLY="true"
+  fi
+fi
+
+# shellcheck disable=SC1091
+source "${CI_DIR}/integration_test_env.sh"
+
+# Run:
+#   - ansible, basic integration
+export FORCE_REPO_UPDATE=false
+
+if [[ "${IMAGE_OS}" == "ubuntu" ]]; then
+    #Must match with run_fetch_logs.sh
+    export CONTAINER_RUNTIME="docker"
+    export EPHEMERAL_CLUSTER="kind"
+else
+    export EPHEMERAL_CLUSTER="minikube"
+fi
+
+git clone "https://github.com/${REPO_ORG}/${REPO_NAME}.git" tested_repo
+cd tested_repo
+git checkout "${REPO_BRANCH}"
+cd ../
+
+if [[ "${REPO_NAME}" == "metal3-dev-env" ]] ||
+    [[ "${REPO_NAME}" == "cluster-api-provider-metal3" ]] \
+    ; then
+    # If we are testing e2e from capm3,
```

Comment:
```suggestion
    # If we are testing e2e from capm3,
```
Is not this test only for ansible tests how we can be testing e2e here ?

Reply:
I mean at the end we will do :

```
echo "Running the tests"

make
make test

```
and this will never run any e2e